Assets/_Project/Scripts/Map/LocationProviderMapInitializer.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Mapbox.Unity.Location;
+using Mapbox.Unity.Map;
+using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.Map
+{
+    public class LocationProviderMapInitializer : MonoBehaviour
+    {
+        [SerializeField] private AbstractMap _map;
+
+        [SerializeField] private LocationFactory _locationFactory;
```

Comment:
Unused field.

Reply:
As we discussed in the DMs I am gonna delete this class and then when we merge the branches I am going to create One class responsible for the entirety of map initialization. For now the resposibility held by the LocationProviderMapUpdater class.